### PR TITLE
Added Redis for persistence with installation/usage info and example code/tests

### PR DESCRIPTION
**Pull request by diegoroyo**
I set up Redis to provide a persistence service for the application. More information can be found on `description.md`.

I also added some example code that tracks the last time you visited the page using Redis along with some basic tests that check if it's working correctly. Redis storage uses the request's IP address as a key and the last visit's time as a value.

### CODE REVIEW

File: src/test/java/es/unizar/webeng/hello/LastVisitUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import static org.hamcrest.Matchers.is;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(LastVisit.class)
+public class LastVisitUnitTest {
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/LastVisitUnitTest.java]: Your code is worthwhile  (reasonable use of Java 8, add a persistent storage, etc.) but contains ideas from previous works and code shown in class, so, in order to obtain a :gift: you should add here a test that explores the scenario of the second visit. 

*helper tip*: Google for `RequestPostProcessor` and `remoteAddr`.

**Comment by fjlopez** [comment]
If you are not willing to do the requested task to get the :gift: answer here asap to merge your code.

**Comment by diegoroyo** [comment]
I added two new unit tests: one checks the 'second visit' scenario and the other checks a scenario where two different users send a request. Thanks for the helper tip, it was pretty useful.

**Comment by fjlopez** [comment]
Resolve the conflict of README.md and I will merge your code (with the :gift:)

**Comment by diegoroyo** [comment]
Done

**Comment by fjlopez** [comment]
Ops, not yet, I merged without realise that you have additional content. My bad. Can you solve it again. I promise not to merge until you fix this.

**Comment by diegoroyo** [comment]
Alright, no problem. Conflicts are solved again.

**Pull request merged**